funding/manager.go

```diff
@@ -957,6 +983,48 @@ func (f *Manager) stateStep(channel *channeldb.OpenChannel,
 	// fundingLocked was sent to peer, but the channel was not added to the
 	// router graph and the channel announcement was not sent.
 	case fundingLockedSent:
+		if channel.ChanType.IsZeroConf() {
+			// We'll need to wait until funding_locked has been
```

Comment:
I would expect this to actually be within the `markedOpen` case handling, since to my reading we've already sent a funding locked message, without first obtaining an alias here.

Reply:
why? i put this here since we have sent funding locked so intuitively we'd be in this state. we'll wait until receipt via the call to `GetAlias`

Reply:
Here we've already sent a fundinglocked with an "our" alias, and are waiting for the peer to send theirs so we can give them a channel_update